codec_specs.md

```diff
@@ -1,21 +1,61 @@
 ---
 layout: default
 ---
-# CodecID
+# Codec Mappings
 
-As an additional resource to this page Haali has created a [list of codec IDs in a PDF](http://haali.su/mkv/codecs.pdf).
+A `Codec Mapping` is a set of attributes to identify, name, and contextualize the format and characteristics of encoded data that can be contained within Matroska Clusters.
```

Comment:
More generally I think it should say that the codec data correspond to the Payload data found in `Block` and `SimpleBlock`,